docs: add release note about github discussions (#1988)
commercetools/merchant-center-application-kit: website/src/releases/2021-01/community-discussions.mdx

---
date: 2021-01-22
title: GitHub Discussions available for application-kit and ui-kit repositories.
description: |
  The GitHub Discussions feature is now available in our open source repositories application-kit and ui-kit.\
  We strongly encourage and recommend to use that as a way of communicating with us (commercetools) and with the community. You can ask questions, bring up ideas, showcase your work, etc.
  More info about GitHub Discussions can be found here: https://docs.github.com/en/discussions
type: announcement
topics:
  - Community
---

[GitHub Discussions](https://docs.github.com/en/discussions) is finally generally available! 🎉

> We have enabled that in our Open Source repositories [application-kit](https://github.com/commercetools/merchant-center-application-kit/discussions/1985) and [ui-kit](https://github.com/commercetools/ui-kit/discussions/1741).

Reviewer: ```suggestion
We have enabled it in a few select Open Source repositories [application-kit](https://github.com/commercetools/merchant-center-application-kit/discussions/1985) and [ui-kit](https://github.com/commercetools/ui-kit/discussions/1741).
```

> We strongly encourage and recommend to use that as a way of communicating with us (commercetools) and with the community. You can ask questions, bring up ideas, showcase your work, etc.

Reviewer: ```suggestion
We strongly encourage and recommend to use that as a way of communicating with us (commercetools) and with the community. You can ask questions, share ideas, showcase your work, etc.
```
